app/src/main/java/com/trainstation/frettirtest2/service/WebViewActivity.java

```diff
@@ -12,7 +12,7 @@
 public class WebViewActivity extends Activity {
 
     private WebView webView;
-    public String url = "http://google.com";
+    private final String URL = "http://google.com";
```

Comment:
see discussion here about have the variable as final and why the name of the variable is upper case --> https://github.com/helgio/frettaApp/pull/2/files#r51474063